docs/reference/remap/functions/redact.cue

```diff
@@ -0,0 +1,94 @@
+package metadata
+
+remap: functions: redact: {
+	category: "String"
+	description: """
+		Redact potentially sensitive data in `value` such as:
+
+		- credit card numbers
+		- custom patterns
+		- (more to come!)
+
+		This can act as a last resort in the case your applications log sensitive data by accident to ensure you do not
+		leak this information into your log storage.
+		"""
+
+	arguments: [
+		{
+			name: "value"
+			description: #"""
+				The value to redact sensitive data from.
+
+				Its behavior differs depending on the type of `value`:
+				- For strings, it simply redacts the sensitive data and returns a new string
+				- For arrays, it redacts the sensitive data in each string element
+				- For objects, it masks the sensitive data in each string value, but not keys
```

Comment:
```suggestion
				Its behavior differs depending on the type of `value`:
				
				- For strings, it simply redacts the sensitive data and returns a new string
				- For arrays, it redacts the sensitive data in each string element
				- For objects, it masks the sensitive data in each string value, but not keys
```

Reply:
👀 the only change I see here is an extra newline, is that right? I'm not sure why the other lines are shown as a diff.

Reply:
Correct